test/node/src/select.test.ts

```diff
@@ -879,16 +879,45 @@ for (const dialect of DIALECTS) {
       })
     }
 
-    if (dialect === 'postgres' || dialect === 'mysql' || dialect === 'mssql') {
```

Comment:
Could also change this to `|| dialect === 'sqlite'` which would make the diff less noisy, but also seemed strange it then includes all the dialects.